pkg/prometheus/promcfg.go

```diff
@@ -1078,6 +1078,14 @@ func (cg *ConfigGenerator) generateProbeConfig(
 
 		// Relabelings for prober.
 		relabelings = append(relabelings, []yaml.MapSlice{
+			{
+				{Key: "source_labels", Value: []string{"__tmp_prometheus_address"}},
```

Comment:
this should be `__address__`, i'll send a follow-up PR because I wanted to tidy up the API documentation anyway.

Reply:
Thanks for your correction.

Reply:
No problem!